### DISCUSSION TITLE
Add verification in flathub

### DISCUSSION THREAD

**Discussion by NamelessGO** [FreeTubeApp/FreeTube#4339] [General]
Since Linux users have access to flatpak on the main website, may the project be validated in flathub? This would inform users that flatpak is associated with the project and foster trust.

https://docs.flathub.org/docs/for-app-authors/verification

https://flathub.org/apps/io.freetubeapp.FreeTube

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Thanks for pointing this out! We will set this up soon. I'll close this discussion when it has been setup.

**Reply by MysteriousOrb2000** [reply]
Any update on this?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
We have planned to add this when we do our next release. No ETA on release though.

**Reply by cassidyjames** [reply]
Adding context: You can [log into Flathub](https://flathub.org/login) with your GitHub account; if you have commit access to the Flatpak manifest, your app will automatically show on your dashboard. If you don't have commit access, that's something we should get set up—I can file an issue on Flathub if that's the case.

Once you're logged in, you're given the verification token for your app, which can then be placed in a plain text file at `https://freetubeapp.io/.well-known/org.flathub.VerifiedApps.txt`. It's a pretty quick process and doesn't technically require making a release.

I'm happy to help out and answer any additional questions!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Closing, https://github.com/flathub/io.freetubeapp.FreeTube/issues/96#issuecomment-1915190123
